src/Video/Buffer/FrameBuffer.hpp

```diff
@@ -19,14 +19,23 @@ namespace Video {
             ~FrameBuffer();
 
             /// Bind this framebuffer as render target.
-            void Bind() const;
+            void Bind();
```

Comment:
A little confusing since we have two bind methods where one is simply called 'Bind'. Maybe 'BindWrite' or 'BindTarget'?